packages/react-app/src/views/Hints.jsx

```diff
@@ -113,7 +113,7 @@ export default function Hints({ yourLocalBalance, mainnetProvider, price, addres
           optionFilterProp="children"
         >
           {listOfTokens.map(token => (
-            <Option key={token.symbol} value={token.symbol}>
+            <Option key={token.address} value={token.symbol}>
```

Comment:
Nice catch! Idk if is to much, but you could even concatenate symbol more address there to even avoid address conflict. (i know looks impossible that to happend). Up to you.

Reply:
I updated to be key={token.address + "_" + token.symbol}